lms/static/sass/_ie.scss

```diff
@@ -72,7 +69,7 @@
     width: 660px;
 
     li.partner {
-      float: left;
+      @include float(left);
       display: block;
       padding: 0;
       width: 220px;
```

Comment:
Do the `header.course-preview` and `hgroup` rules a bit below this need to have the bi-app syntax for the left and right variables declared (honestly not sure but i think you likely no best)